packages/flutter/lib/src/widgets/navigator.dart

```diff
@@ -3914,6 +3923,7 @@ class NavigatorState extends State<Navigator> with TickerProviderStateMixin, Res
     ];
   }
 
+  _RouteEntry? _lastEntry;
```

Comment:
could this get a more descriptive name? Presumably, this is the `_lastTopmostRoute` or something like that?